services/QuillLMS/app/models/concerns/user_flagset.rb

```diff
@@ -71,9 +70,10 @@ module UserFlagset
           Flags::PRIVATE =>           { display_name: 'Private' }
         }
       },
-
     }
 
+    DEFAULT_ACTIVITY_SEARCH_CACHE_KEY = 'default_activity_search'
```

Comment:
No objection to this being a constant, but is this the right file for it to be defined in?  Just checking.

Reply:
I put it in that concern because the other cache keys were populated via iterating over UserFlagset:
![Screenshot 2023-08-07 at 4 35 08 PM](https://github.com/empirical-org/Empirical-Core/assets/2057805/3b4a4ee0-d5ac-4197-834c-40448e8168dc)

However, on second thought I think it belongs in activity.rb.